tests/foreman/cli/test_remoteexecution.py

```diff
@@ -133,6 +135,10 @@ def test_positive_run_default_job_template(
         )
         assert 'Exit' in out
         assert 'Internal Server Error' not in out
+        #  SAT-29062
+        hour = search[0]['started-at'].split(" ")[1].split(":")[0]
+        search = module_target_sat.cli.Task.list_tasks({'search': f'started_at > {hour}:00 '})
+        assert task['id'] in [t['id'] for t in search], "task not found in expected time range"
```

Comment:
I suggest to add a test that it's NOT included in search for "started_at > {hour+1}:00"